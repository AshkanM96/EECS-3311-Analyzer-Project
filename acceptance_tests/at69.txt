disjoin
disjoin
disjoin
boolean_constant(true)
boolean_constant(false)
boolean_constant(true)
disjoin
boolean_constant(true)
disjoin
integer_constant(-2)
boolean_constant(false)
type_check
evaluate
