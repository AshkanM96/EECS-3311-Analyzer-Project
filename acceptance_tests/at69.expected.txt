  Expression currently specified: ?
  Report: Expression is initialized.
->disjoin
  Expression currently specified: (? || nil)
  Report: OK.
->disjoin
  Expression currently specified: ((? || nil) || nil)
  Report: OK.
->disjoin
  Expression currently specified: (((? || nil) || nil) || nil)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (((True || ?) || nil) || nil)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (((True || False) || ?) || nil)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (((True || False) || True) || ?)
  Report: OK.
->disjoin
  Expression currently specified: (((True || False) || True) || (? || nil))
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (((True || False) || True) || (True || ?))
  Report: OK.
->disjoin
  Expression currently specified: (((True || False) || True) || (True || (? || nil)))
  Report: OK.
->integer_constant(-2)
  Expression currently specified: (((True || False) || True) || (True || (-2 || ?)))
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (((True || False) || True) || (True || (-2 || False)))
  Report: OK.
->type_check
  Expression currently specified: (((True || False) || True) || (True || (-2 || False)))
  Report: (((True || False) || True) || (True || (-2 || False))) is not type-correct.
->evaluate
  Expression currently specified: (((True || False) || True) || (True || (-2 || False)))
  Report: Error (Expression is not type-correct).
